src/content/test/text-legibility/reflow.tsx

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { create, React } from '../../common';
+
+export const infoAndExamples = create(({ Markup }) => (
+    <>
+        <h1>Reflow</h1>
+        <p>Text content must be visible without having to scroll both vertically and horizontally.</p>
+
+        <h2>Why it matters</h2>
+        <p>
+            Having to scroll in two directions is difficult for everyone. Having to scroll in the direction of reading makes reading
+            especially difficult for people with certain disabilities, including:
+        </p>
+        <ul>
+            <li> People with low vision, who are more likely to need enlarged text in a single column</li>
+            <li> People with reading disabilities that make it difficult to visually track long lines of text</li>
+            <li> People with motor disabilities who find scrolling difficult</li>
+        </ul>
+        <h2>How to fix </h2>
+        <p>
+            Use responsive web design to ensure page content reflows to fit within the window whenever the viewport narrows or the text size
+            changes.
+        </p>
+        <ul>
+            <li>Good: Do not require users to scroll in two directions.</li>
+            <li>
+                Better: Do not require users to scroll in the direction of reading:
+                <ul>
+                    <li>the script is read horizontally (left-to-right or right-to-left), text should scroll vertically.</li>
```

Comment:
These items read a little awkwardly; are they intended to start with "**if** the script"?

Reply:
yes, the converter cut that out accidentaly